rubicon_ml/repository/base.py

```diff
@@ -260,19 +302,12 @@ def get_experiments(self, project_name):
             The experiments logged to the project with name
             `project_name`.
         """
-        experiment_metadata_root = self._get_experiment_metadata_root(project_name)
-
         try:
-            experiment_metadata_paths = self._ls_directories_only(experiment_metadata_root)
-            experiments = [
-                domain.Experiment(**json.loads(metadata))
-                for metadata in self._cat_paths(experiment_metadata_paths)
-            ]
-            experiments.sort(key=lambda e: e.created_at)
+            experiment_metadata_root = self._get_experiment_metadata_root(project_name)
         except FileNotFoundError:
             return []
```

Comment:
`_get_*_metadata_root` isn't ever gonna raise a `FileNotFoundError` in any of these anymore